service/service.go

```diff
@@ -87,7 +89,10 @@ func GetRouter(as *APIServer, uis *UIServer) (http.Handler, error) {
 	uiService := uis.GetServiceApp()
 	apiService := as.GetServiceApp()
 
+	router := mux.NewRouter().UseEncodedPath()
+	router.Use(otelmux.Middleware("evergreen"))
```

Comment:
Should this be more specific, something like `evergreen-rest`?

Reply:
I think it's a really generic router, though. Like this router handles just about all of the incoming traffic from REST, GraphQL, the old UI, etc.

Reply:
Sound good!